include/modem/lte_lc.h

```diff
@@ -214,21 +214,27 @@ enum lte_lc_evt_type {
 	 */
 	LTE_LC_EVT_NW_REG_STATUS,
 
+#if defined(CONFIG_LTE_LC_PSM) || defined(CONFIG_ZTEST) || defined(CONFIG_UNITY) || \
+	defined(__DOXYGEN__)
 	/**
 	 * PSM parameters provided by the network.
 	 *
 	 * The associated payload is the @c lte_lc_evt.psm_cfg member of type
 	 * @ref lte_lc_psm_cfg in the event.
 	 */
 	LTE_LC_EVT_PSM_UPDATE,
+#endif
 
+#if defined(CONFIG_LTE_LC_EDRX) || defined(CONFIG_ZTEST) || defined(CONFIG_UNITY) || \
+	defined(__DOXYGEN__)
```

Comment:
You should get rid of `defined(__DOXYGEN__)` by adding `CONFIG_LTE_LC_EDRX=y` into `PREDEFINED` list in `/home/tor1/work/ncs/nrf/doc/nrf/nrf.doxyfile.in`.

This is what we did in Location library and seems modem info and libmodem has their flags in there too.